nixos/modules/system/boot/systemd.nix

```diff
@@ -435,6 +435,24 @@ in
             type = service.serviceConfig.Type or "";
             restart = service.serviceConfig.Restart or "no";
             hasDeprecated = builtins.hasAttr "StartLimitInterval" service.serviceConfig;
+
+            # We exclude systemd services in nixpkgs to reduce noise,
+            # so we can notice restart policy warnings associated with our services
+            isExternalService = !(elem name serviceNames) && !(any (pref: lib.hasPrefix pref name) extraServiceNames);
+            # File serviceNames is created manually using
+            # grep -RPho '(?<=systemd\.services\.)(?!.*\$)(.*?)(?=\.| )' ./nixos/modules --include=\*.nix | sort | uniq | sed 's/\"//g' > ./nixos/modules/system/boot/serviceNames
+            serviceNames = lib.splitString "\n" (builtins.readFile ./serviceNames);
+            # Some services are missing from the serviceNames, so we need to add them separately
+            extraServiceNames = [
+              "network-"
+              "nix-optimise"
+              "mount-pstore"
+              "wireguard-"
+              "acme-"
+              "mdmonitor"
+              "iodined"
+              "copr-cli"
```

Comment:
Hmm, I think that's not an actual service and just a leftover that is relevant to https://github.com/serokell/vault-secrets/issues/25 :sweat_smile:

Reply:
Oh really, thank you